modules/ROOT/pages/cloudhub-faq.adoc

```diff
@@ -156,7 +174,7 @@ If you have a paid plan, in addition to the forums, you also have access to the
 . When finished, click *Submit*. The portal generates a unique case number and refer you to possibly relevant areas of the knowledge base that you can browse while you await a response to your case.
 
 
-=== Proactive Monitoring
+== Proactive Monitoring
 
 The CloudHub platform is monitored 24 hours a day, seven days a week by automated systems. In the event of any issue affecting the health and operation of the CloudHub infrastructure, our dedicated operations team is notified and will respond immediately to diagnose and correct it. This 24x7 monitoring covers the entire CloudHub platform, benefiting all CloudHub users, free and paying. The status of CloudHub services and upcoming maintenance can be found at http://status.mulesoft.com[`status.mulesoft.com`].
```

Comment:
[Some texts won't show the correx when copied...I don't know why. So these are individual:

seven days a week by > 7 days a week, by
will respond immediately > responds immediately